gnd/src/main/java/com/google/android/gnd/system/PermissionsManager.java

```diff
@@ -36,15 +38,25 @@
 
   @Inject
   public PermissionsManager(Application app, ActivityStreams activityStreams) {
-    context = app.getApplicationContext();
+    this.context = app.getApplicationContext();
     this.activityStreams = activityStreams;
   }
 
+  /**
+   * Asynchronously requests the app be granted the specified permission. If the permission has
```

Comment:
"the specified permission" -> "the specified {@param permission}"?

Reply:
I meant "the permission" in general, otherwise we might say "synchronously requests the app be granted {@param permission}".

Reply:
If you don't mind I'll merge as-is to avoid needing reapproval. Happy to reword if you feel it should be changed!